remix.init/index.js

```diff
@@ -0,0 +1,19 @@
+const fs = require("fs");
+const path = require("path");
+
+module.exports = ({ isTypeScript, rootDirectory }) => {
+  if (isTypeScript === false) {
```

Comment:
[sand] Nitpick. No need to explicitly check agains false.

```suggestion
  if (!isTypeScript) {
```

Or you could exit early.

```javascript
module.exports = ({ isTypeScript, rootDirectory }) => {
  if (isTypeScript) {
    return;
  }

  let remixConfigPath = path.join(rootDirectory, "remix.config.js");
  let remixConfig = require(remixConfigPath);
  remixConfig.server = "./server.js";

  let newConfig = `
/**
 * @type {import('@remix-run/dev').AppConfig}
 */
module.exports = ${JSON.stringify(remixConfig, null, 2)}
`.trim();

  fs.writeFileSync(remixConfigPath, newConfig);
};
```